lib/IntermediateRepresentation/builder.cc

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "builder.h"
+#include <iostream>
```

Comment:
LLVM restrictions on static objects with constructors preclude the use of `<iostream>` in permanent code.  The uses already in f18 are mostly for scaffolding debugging and will have to get extracted.

Reply:
Fixed